static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -44,6 +46,23 @@ function BreadcrumbItem({
     (e: React.MouseEvent<HTMLElement>) => onClick?.(crumb, e),
```

Comment:
wait, can we combine these click handlers?

Why scope clicks to only the breadcrumb icon

Reply:
> What's the correct ticket for this PR? The linked #36803 looks like it's talking about the Timeline instead.

Err duhh yeah that's the right ticket I just did the wrong thing. I'll add handler for the timeline as well.

> wait, can we combine these click handlers?

Totally I read the icon part specifically as part of the ticket thats why I did it that way